spec.md

```diff
@@ -33,7 +33,7 @@ effects on the host application.
 
 Starlark is intended to be simple. There are no user-defined types, no
 inheritance, no reflection, no exceptions, no explicit memory management.
-Execution is finite. The language does not allow recursion or unbounded loops.
+The language does not allow unbounded loops.
```

Comment:
The old wording seems fine to me. In particular, it's worth calling out recursion as many programmers don't realize that loop-recursion and function-recursion are isomorphic.

Reply:
Reverted.